src/modules/search-result/index.js

```diff
@@ -1 +1 @@
-export SearchResult from './search-result'
+export default from './search-result'
```

Comment:
Honestly, I don't know why but without this change the import `import {SearchResult} from '@oacore/design'` doesn't work.

Reply:
The first one is named. I think `./modules/index` exports it in some specific way kind of `export SearchResult from '...'` what exports default which wasn't exported here where you fixed it.